### ISSUE TITLE
BundleResource usage fails build with CS1508

### ISSUE THREAD

**Issue by LeadAssimilator** [xamarin/Xamarin.Legacy.Sdk#57]
XMI 16.4.0.15,.NET 6.0.201,7.0.401 w/ 0.2.0-alpha4

When building an iOS binding project targeting xamarin.ios10 with one or more BundleResource items, build fails with CS1508.

Workaround:

```xml
  <Target Name="_RemoveEmbeddedResourceDuplicates" BeforeTargets="_GenerateCompileInputs">
    <ItemGroup>
      <_DistinctEmbeddedResource Include="@(EmbeddedResource->Distinct())" />
      <EmbeddedResource Remove="@(EmbeddedResource)" />
      <EmbeddedResource Include="@(_DistinctEmbeddedResource)" />
    </ItemGroup>
  </Target>
```

**Comment by jonathanpeppers** [comment]
Can you share a `.binlog` of this issue? https://aka.ms/binlog

Is something including all `EmbeddedResource` / `BundleResource` items twice?

**Comment by LeadAssimilator** [comment]
_PackLibraryResources and _CreateEmbeddedResources are both adding the same items.  If I omit BundleResource items then neither targets add any items to EmbeddedResource (or run, don't remember), but they both do as soon as I add BundleResource items back.

I can eventually get a binlog, but because they are still not properly sanitized it will be a while - it should be trivial to repro though in the interim.

**Comment by jonathanpeppers** [comment]
Yeah, it's just not going to be a high priority for us to look at this. We mostly use this for building AndroidX libraries, and Xamarin goes out of support in May (as of now). You may need to send us a pull request, if you have an idea for a fix, thanks!

**Comment by LeadAssimilator** [comment]
Yeah, but priority or not, you should still triage it at least.  And that date is more than 6mos away, and will probably get pushed back (or so I hope) because net 6/7/8 are still broken in countless ways preventing migration, it is a huge amount of work to update and one dependency not being updated is all it takes to ruin everything.  Having tooling to ease migration is actually pretty important which is what I thought this project was attempting to provide on the dependency front...

Here is the binlog - [msbuild.zip](https://github.com/xamarin/Xamarin.Legacy.Sdk/files/12679966/msbuild.zip)

**Comment by jonathanpeppers** [comment]
We tried to put a disclaimer on this project, just fyi:

![image](https://github.com/xamarin/Xamarin.Legacy.Sdk/assets/840039/f17db91e-d2d1-4420-ad14-84275a115968)

**Comment by LeadAssimilator** [comment]
Not fully supported sure, but BundleResource - that is pretty basic...

**Comment by jonathanpeppers** [comment]
Yes, we made this primarily for Android, and I believe a contributor added Mac & iOS.
